### PR TITLE
A subset of the first batch of Kathleen's comments addressed in-text

### PR DESCRIPTION
**Pull request by henkbirkholz**
* based on: Based on https://mailarchive.ietf.org/arch/msg/rats/2rwmcm7pQICXN8lzlX21lEkb9Is/
* used "integrity" instead of "hygine" for now
* used "evaluate" instead of "verify" for now

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -106,8 +106,8 @@ such as roles, device composition, topological models, and appraisal is vital fo
 semantic interoperability across solutions and platforms involving multiple vendors and providers.
 
 Amongst other things, this document is about trust and trustworthiness.
-Trust is a decision being made. Trustworthiness is a quality that is
-assessed via evidence created. This is a subtle difference and being
+To trust is a decision that spans beyond the technical assertions. Trustworthiness is a quality that is
```
Comment (nedmsmith) [inline draft-ietf-rats-architecture.md:109]: The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.

Comment (mcr) [reply draft-ietf-rats-architecture.md:109]: > The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.

I'm unclear if you are agreeing or disagreeing here.

Comment (thomas-fossati) [reply draft-ietf-rats-architecture.md:109]: @mcr: I think this is the replacement text that Ned suggests to use.

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -227,7 +228,10 @@ Relying Party:
 This is a generalization of the ML model use case above, where
 the data can be any highly confidential data, such as health data
 about customers, payroll data about employees, future business plans, etc.
-Attestation is desired to prevent leaking data to compromised devices.
+An assessment of system integrity is made against a set of policies
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:231]:
```suggestion
An assessment of system state is made against a set of policies
```

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (mcr) [inline draft-ietf-rats-architecture.md:109]:
```suggestion
The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.
Trustworthiness is a quality that is
```

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -106,8 +106,9 @@ such as roles, device composition, topological models, and appraisal is vital fo
 semantic interoperability across solutions and platforms involving multiple vendors and providers.
 
 Amongst other things, this document is about trust and trustworthiness.
-Trust is a decision being made. Trustworthiness is a quality that is
-assessed via evidence created. This is a subtle difference and being
+The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.
+Trustworthiness is a quality that is
+assessed via evidence. This is a subtle difference and being
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:111]: Trust is a choice one makes about another system.   Trustworthiness is a quality about the other system that can be used in making one's decision to trust it or not.

**Pull request merged**